### ISSUE TITLE
Not working with dedicated/multiplayer

### ISSUE THREAD

**Issue by narrowsoul** [Jeroen-Notenbomer/Limited-Arsenal#1]
You're probably well aware of this, but I can't get it to work in multiplayer. I can load items to crate and they show up just fine for me, but not anyone else connected to the server.

**Comment by TacticalLolicon** [comment]
Same problem, but very excited to see this wonderful script standalone!

**Comment by Jeroen-Notenbomer** [comment]
Yes its not a complete script, but with my last update it seems to be working with mp again.
